analytic_tag_dimension/models/analytic_dimension_line.py

```diff
@@ -10,6 +11,32 @@ class AnalyticDimensionLine(models.AbstractModel):
     _description = "Analytic Dimension Line"
     _analytic_tag_field_name = "analytic_tag_ids"
 
+    @api.model
+    def get_model_names(self):
+        return [
+            "account.move.line",
+            "account.analytic.line",
+            "account.invoice.report",
+        ]
+
+    def _reflect(self):
+        super()._reflect()
+        if (
+            self._inherit != self._name
+            and self._inherit in self.get_model_names()
```

Comment:
Why are you limiting to these models? Any children model should have it, isn't it?